suite-native/graph/src/components/Graph.tsx

```diff
@@ -25,12 +28,24 @@ const graphStyle = prepareNativeStyle(_ => ({
     width: '100%',
 }));
 
-export const Graph = ({ points = [], loading = false }: GraphProps) => {
-    const { applyStyle } = useNativeStyles();
+const placeholderPoints = [
+    {
+        date: new Date(0),
+        originalDate: new Date(),
+        value: 0,
+    },
+];
```

Comment:
Just question why to have this here? Maybe write a comment

Reply:
Not sure it was here before, I only moved it before component. It was probably some edge case fallback from @bouchja1

I will try to remove it.